format/FormatSER.py

```diff
@@ -32,37 +31,36 @@ def __init__(self, image_file, **kwargs):
 
     @staticmethod
     def understand(image_file):
-
         try:
-            tag = FormatSER.open_file(image_file, "rb").read(14)
+            with FormatSER.open_file(image_file, "rb") as fh:
+                tag = fh.read(14)
         except IOError:
             return False
 
         # File should be little endian
-        if not tag[0:2] == "II":
+        if tag[0:2] != b"II":
             return False
 
         # SeriesID: 0x0197 indicates ES Vision Series Data File
-        if not struct.unpack("<H", tag[2:4])[0] == 407:
+        if struct.unpack("<H", tag[2:4])[0] != 407:
             return False
 
         # SeriesVersion: 0x0210 or 0x220
-        if not struct.unpack("<H", tag[4:6])[0] in [528, 544]:
+        if struct.unpack("<H", tag[4:6])[0] not in (528, 544):
             return False
 
         # DataTypeID: 0x4122 if elements are 2D arrays
-        if not struct.unpack("<I", tag[6:10])[0] == 16674:
+        if struct.unpack("<I", tag[6:10])[0] != 16674:
             return False
 
         # TagTypeID: 0x4142 or 0x4152
-        if not struct.unpack("<I", tag[10:])[0] in [16706, 16722]:
+        if struct.unpack("<I", tag[10:])[0] not in (16706, 16722):
```

Comment:
Since we are testing for hex values, better to use hex integer constants (appreciate unchanged in the pR; campsite rules)

Reply:
I didn't even see that when refactoring, so very valid point.